cpp/examples/abm_minimal.cpp

```diff
@@ -165,10 +148,15 @@ int main()
     mio::abm::close_social_events(t_lockdown, 0.9, world.parameters);
 
     auto t0   = mio::abm::TimePoint(0);
-    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(30);
+    auto tmax = mio::abm::TimePoint(0) + mio::abm::days(10);
     auto sim  = mio::abm::Simulation(t0, std::move(world));
 
     sim.advance(tmax);
 
-    write_results_to_file(sim);
+    std::ofstream outfile("abm_minimal.txt");
+    sim.get_result().print_table({"S", "E", "I_NS", "I_Sy", "I_Sev", "I_Crit", "R", "D"}, 7, 4, outfile);
```

Comment:
I would suggest moving some of the comments in write_results_to_file() to here to explain what these characters mean.

Reply:
Done

Reply:
Thanks.
I also forgot here to ask if we need time(t) in the result as well. Otherwise, please delete the relevant line and adjust the number of rows to 8.

Reply:
In short: No, time does not need to be listed, but the number of columns is still nine.
The longer answer: print_table is a TimeSeries member function, hence "Time" is always the first column with exactly that label. All other columns can be labeled using this string vector (the default is #+the column index, starting at 1).

Reply:
Yes. Then I suggest a small adjustment above so the naming follows what is in the output file.

Reply:
I think i added what you suggested.